frontend/src/containers/RecipePage/Recipe.css

```diff
@@ -0,0 +1,154 @@
+.overall-container {
+  margin: 0px auto;
+  padding: 0px;
+}
+
+.user-edit-button {
+  display: inline-block;
+  float: right;
+  margin: 20px 26px;
+  border: 3px solid #ffa616;
+  border-radius: 13%;
```

Comment:
I think you'll want this border radius to be higher to make them fully rounded, like the other buttons on the site. Right now they're a little angular.